mautic/mautic
# [UX] No data found message for dashboard charts

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR returns a message "No data found" when a chart is empty instead of just leaving everything blank (which seemed broken/faulty/buggy).

![image](https://github.com/user-attachments/assets/7a1ff63b-28f0-4ac9-8d9a-e19ec8d862cf)






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the Dashboard and check the widgets that don't have data
